### ISSUE TITLE
How to set Background Image in Template Constants ?

### ISSUE THREAD

**Issue by ds2k5** [benjaminkott/bootstrap_package#946]
Hi,
how to set background image in the template Constants ?

typo3 10.4.9 and bootstrap package 11.0.2

thanks

**Comment by gilbertsoft** [comment]
There are several ways to use background images: some content elements have background images or you can add it via CSS e.g. by using a Sitepackage or an inline CSS.

**Comment by ds2k5** [comment]
Ok thanks, did I understand right that a constant:   "background_image" not exists like "backgroud_color" for the hole page ?

**Comment by gilbertsoft** [comment]
Yes, I'm not aware of this constant. It's also not simple solvable with one constant. As said, use your Sitpackage to change this.
